src/sentry/static/sentry/app/components/eventOrGroupExtraDetails.jsx

```diff
@@ -3,8 +3,8 @@ import React from 'react';
 import createReactClass from 'create-react-class';
 import {Link} from 'react-router';
 import styled from 'react-emotion';
-import {Flex, Box} from 'grid-emotion';
 
+import {Flex, Box} from './grid';
```

Comment:
here's an optional suggestion for the studio audience: I've always disliked the name `Box`. When I first saw it, I thought it was a panel-type thing. A div is not a box and you rarely see `box` as a className...so I was like...what is this?

Maybe `<Container>`? `<Wrapper>`? `<BlockElement>`? `<Div>`?